### ISSUE TITLE
Starting cluster on 3.1.x fails

### ISSUE THREAD

**Issue by imlucas** [christkv/mongodb-tools#1]
```
Error: Command failed: /bin/sh -c mongos --port 50000 --pidfilepath /Users/lucas/.mongodb/data/cluster/mongos-50000.pid --logpath /Users/lucas/.mongodb/cluster.log/mongos-50000.log --configdb localhost:35000
BadValue Must have either 3 node old-style config servers, or a single server replica set config server
try 'mongos --help' for more information

    at ChildProcess.exithandler (child_process.js:751:12)
    at ChildProcess.emit (events.js:110:17)
    at maybeClose (child_process.js:1015:16)
    at Socket.<anonymous> (child_process.js:1183:11)
    at Socket.emit (events.js:107:17)
    at Pipe.close (net.js:485:12)
```

**Comment by imlucas** [comment]
@christkv :heart:

**Comment by christkv** [comment]
fixed an pushed to 1.0.6
